framework/source/class/qx/ui/embed/Iframe.js

```diff
@@ -335,7 +380,11 @@ qx.Class.define("qx.ui.embed.Iframe",
 
       if (iframeSource != this.getSource())
       {
-        qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        if ( qx.core.Environment.get("browser.name") != "edge" &&
+             qx.core.Environment.get("browser.name") != "ie" )
+        {
+          qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        }
```

Comment:
Yes and No... the stop() method is not available in "Internet Explorer".
To **remove** the **Mutation Event** is all this issue is about!